mesheryctl/pkg/utils/formatter.go

```diff
@@ -36,3 +41,62 @@ func SetupMeshkitLogger(debugLevel bool, output io.Writer) {
 	}
 	Log = logger
 }
+
+type Paginator struct {
+	data                 []string
+	lastVisibleItemIndex int
+	whiteBgPrinter       *color.Color
+	userMsg             string
+}
+
+func NewPaginator(header []string, rows [][]string) *Paginator {
+	var buf bytes.Buffer
+	renderedTableString := writeTableToBuffer(&buf, header, rows)
+	lastVisibleItemIndex := 26
+	data := strings.Split(renderedTableString, "\n")
+	userMsg := "Press ↓ to load more items, esc and Ctrl+C to quit"
```

Comment:
Please change to spacebar as both the keystroke and the help description.
```suggestion
	userMsg := "Press spacebar to load more. Press ESC or Ctrl+C to stop."
```

Reply:
@zakisk in the future, please accept these suggestions so that I don't have to go separately confirm that you incorporated the feedback.